sqlmesh/core/dialect.py

```diff
@@ -759,3 +763,17 @@ def find_tables(expression: exp.Expression, dialect: DialectType = None) -> t.Se
         for table in scope.tables
         if not isinstance(table.this, exp.Func) and exp.table_name(table) not in scope.cte_sources
     }
+
+
+def transform_values(
+    values: t.Tuple[t.Any, ...], columns_to_types: t.Dict[str, exp.DataType]
+) -> t.Iterator[t.Any]:
+    """Perform transformations on values given columns_to_types.
+
+    Currently, the only transformation is wrapping JSON columns with PARSE_JSON().
+    """
+    for value, col_type in zip(values, columns_to_types.values()):
+        if col_type == exp.DataType.build("json"):
```

Comment:
How about factoring out the `exp.DataType.build("json")` call to avoid rebuilding it over and over again?

Reply:
That's a good idea. I'll do that.